STM32Cube/Tasks/vn_handler.c

```diff
@@ -222,7 +226,7 @@ void vnIMUHandler(void *argument)
 								send3VectorStateCanMsg_double(time_startup, posEcef.c,STATE_POS_X); //position
 								send3VectorStateCanMsg_float(time_startup, velEcef.c,STATE_VEL_X); //velocity
 								send3VectorStateCanMsg_float(time_startup, linAccelEcef.c,STATE_ACC_X); //acceleration
-								send3VectorStateCanMsg_float(time_startup, yprAngles.c,STATE_ANGLE_YAW); //angle; TODO: this is also sent by OUR state estimation, so we should expand the enum in canlib to distinguish
+								//send3VectorStateCanMsg_float(time_startup, yprAngles.c,STATE_ANGLE_YAW); //angle; TODO: this is also sent by OUR state estimation, so we should expand the enum in canlib to distinguish
```

Comment:
becuase state est takes care of it?

Reply:
This was to resolve the ID conflict. Now that that has been fixed I will change it back